backend/app/usecase/repository/shortlink_fake.go

```diff
@@ -105,6 +105,20 @@ func (s ShortLinkFake) UpdateShortLink(oldAlias string, shortLinkInput entity.Sh
 	}, nil
 }
 
+// DeleteShortLink deletes an existing user short link from in memory data store.
+func (s ShortLinkFake) DeleteShortLink(alias string) error {
+	if alias == "" {
+		return errors.New("empty alias")
+	}
+	_, ok := s.shortLinks[alias]
+	if !ok {
+		return errors.New("alias does not exist")
```

Comment:
```suggestion
		return errors.ErrEntryNotFound
```
Or in your current semantics, it's short link not found.